File: src/01/01prime.go
package main

import "fmt"

func check(s int) int {
	count := 0
	for i := 1; i < s; i++ {
		if s%i == 0 {
			count++
		}
	}
	if count == 2 {
		return 1
	} else {
		return 0
	}
}

func main() {
	n := 859433
	count := 0

	for i := 1; i < n; i++ {
		if check(i) == 1 {
			count++
		}
	}
	if count > 0 {
		fmt.Printf("%d は素数です。%d 番目です。\n", n, count)
	} else {
		fmt.Printf("%d は素数ではありません。\n", n)
	}
}

File: src/01/01prime.c
#include<stdio.h>

int check(int s){
  int count = 0;
  for(int i = 1; i <= s; i++){
    if(s % i == 0)count++; 
  }
  if(count == 2)return 1;
  return 0;
}

int main(){
  int n = 859433, count = 0;

  for(int j = 1; j <= n; j++){
    if(check(j) == 1){
      count++;
      printf("%d\n", count);
    }
  }
  printf("%d\n", count);
}